client/Lykke.Service.ChainalysisProxy.Client/AutofacExtension.cs

```diff
@@ -22,16 +24,34 @@ public static void RegisterChainalysisProxyClient(this ContainerBuilder builder,
                 .SingleInstance();
         }
 
+        public static void RegisterChainalysisProxyClient(this ContainerBuilder builder, string serviceUrl, ILogFactory logFactory, int timeout)
+        {
+            if (builder == null) throw new ArgumentNullException(nameof(builder));
+            if (serviceUrl == null) throw new ArgumentNullException(nameof(serviceUrl));
+            if (logFactory == null) throw new ArgumentNullException(nameof(logFactory));
+            if (string.IsNullOrWhiteSpace(serviceUrl))
+                throw new ArgumentException("Value cannot be null or whitespace.", nameof(serviceUrl));
+
+            builder.RegisterType<ChainalysisProxyClient>()
+                .WithParameter("serviceUrl", serviceUrl)
+                .WithParameter("log", logFactory.CreateLog(nameof(ChainalysisProxyClient)))
```

Comment:
pass ILogFactory to the ctor, not ILog. Mark ChainalysisProxyClient ctor with ILog as obsolete and add new one with ILogFactory